west.yml

```diff
@@ -68,6 +70,10 @@ manifest:
     - name: mbedtls
       revision: ca32746072ce3381f1c9ae46ba6cd34c69f8c0ee
       path: modules/crypto/mbedtls
+    - name: mcuboot
+      revision: 358ca1ae4fa82b97742c5b1a1822d2d9b114b582
+      path: mcuboot
```

Comment:
path defaults to name, so this is redundant, just FYI

Reply:
didn't know, thanks for the tip

Reply:
moved to `applications/mcuboot` path